src/main_app/db/services/charts_and_templates_service.py

```
@@ -0,0 +1,94 @@
+        try:
+            query = (
+                self.session.query(
+                    OwidChartRecord,
+                    TemplateRecord.id.label("template_id"),
+                    TemplateRecord.title.label("template_title"),
+                )
+                .outerjoin(TemplateRecord, TemplateRecord.slug == OwidChartRecord.slug)
+                .order_by(OwidChartRecord.chart_id.asc())
+            )
+            result = query.all()
+        except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)